cmd/sovereignnode/runType/argRunTypeComponentsCreator.go

```diff
@@ -0,0 +1,40 @@
+package runType
+
+import (
+	"fmt"
+
+	"github.com/multiversx/mx-chain-go/config"
+	"github.com/multiversx/mx-chain-go/factory/runType"
+	"github.com/multiversx/mx-chain-go/sovereignnode/dataCodec"
+	"github.com/multiversx/mx-chain-go/sovereignnode/incomingHeader"
+
+	"github.com/multiversx/mx-sdk-abi-incubator/golang/abi"
+)
+
+// CreateSovereignArgsRunTypeComponents creates the args for run type component
+func CreateSovereignArgsRunTypeComponents(
```

Comment:
This func is a bit "weird".
You create args using args...
Perhaps those extra comps(like codec) should be included in those coreSovComps as discussed before and this func should not exist.

Reply:
Added this update in `MX-12902` Task for RunTypeCoreComponents